daprdocs/content/en/developing-applications/ides/vscode/vscode-how-to-debug-multiple-dapr-apps.md

```diff
@@ -0,0 +1,173 @@
+---
+type: docs
+title: "How-To: Debug multiple Dapr applications"
+linkTitle: "How-To: Debug multiple Dapr applications"
+weight: 40000
+description:  "How to configure Dapr for multiple applications"
+---
+
+As your project grows you will experience a need to configure VS Code to handle multiple Dapr applications to debug your project. This guide will offer you a simple to follow explanation of the type of changes you will need
+in order to ensure that your debugging experience is consistent.
+
+To follow along with the breakdown of configuration features please setup the [Hello World Quickstart Project](https://github.com/dapr/quickstarts/tree/v1.0.0/hello-world) 
+
+## Step 1: Configure launch.json
+This file contains information regarding the configurations you run during the debugging process. For the Hello World project, you will have two applications running along side 2 Dapr instances.
+Each SDK supported will require its own tweaks for the launching of the program but each configuration will contain a Daprd run task and a Daprd stop task.
+
+
+
+### NodeJS
+
+```json
+{
+    "type": "pwa-node",
+    "request": "launch",
+    "name": "Nodeapp with Dapr",
+    "skipFiles": [
+        "<node_internals>/**"
+    ],
+    "program": "${workspaceFolder}/app.js",
+    "preLaunchTask": "daprd-debug-node",
+    "postDebugTask": "daprd-down-node"
+}
+```
+
+### Python
+
+```json
+{
+    "type": "python",
+    "request": "launch",
+    "name": "Pythonapp with Dapr",
+    "program": "${workspaceFolder}/app.py",   
+    "console": "integratedTerminal",
+    "preLaunchTask": "daprd-debug-python",
+    "postDebugTask": "daprd-down-python"
+}
+```
+
+The 3 main parameters each configuration will need is a `request`, `type` and `name`. In these scenarios we need only the `launch` request as we will be launching each application and connecting to a running Daprd instance which we will launch right before the application itself.
```

Comment:
The last sentence here is a bit confusing. Can you simplify? Lets remember some of our readers are not native English speakers